### PR TITLE
migrating from npm to yarn

### PR DESCRIPTION
**Pull request by alex-krasn**
We're migrating from ***npm*** to ***yarn***

Yarn official documentation: https://classic.yarnpkg.com/en/docs/migrating-from-npm

For migration you will need:
- delete ***node_modules*** folder, 
 - and run:
``` yarn import``` - after that delete ***package-lock.json ****,
``` yarn install```,
```yarn add react-scripts typescript```,
also you might need to run:``` yarn upgrade  ```.

#302

### CODE REVIEW

**Comment by kunzms** [comment]
We also need to change pipeline

File: azure-pipelines.yml [hunk 1]
```diff
@@ -16,17 +16,17 @@ steps:
   displayName: 'Install Node.js'
 
 - script: |
-    npm install
-    npm run tslint
+    yarn install
+    yarn run tslint
```
Comment (kunzms) [inline azure-pipelines.yml:20]: for yarn, I think "yarn tslint" is enough

File: azure-pipelines.yml [hunk 2]
```diff
@@ -16,17 +16,17 @@ steps:
...
     if [ $? -ne 0 ]; then
         exit 1
     fi
-    npm run compile 
+    yarn run compile
```
Comment (kunzms) [inline azure-pipelines.yml:24]: yarn compile

File: azure-pipelines.yml [hunk 3]
```diff
@@ -16,17 +16,17 @@ steps:
...
     if [ $? -ne 0 ]; then
         exit 1
     fi
-    npm run build
+    yarn run build
```
Comment (kunzms) [inline azure-pipelines.yml:28]: yarn build

File: azure-pipelines.yml (hunk 2 above)
Comment (alex-krasn) [reply azure-pipelines.yml:24]: sure let me check, I never use yarn before

Comment (alex-krasn) [reply azure-pipelines.yml:24]: you're right - will fix it in a second (working both ways though)

File: azure-pipelines.yml (hunk 3 above)
Comment (alex-krasn) [reply azure-pipelines.yml]: done

**Comment by alex-krasn** [comment]
closes issue #302

**Pull request merged**